message/model.json

```diff
@@ -3,20 +3,26 @@
     "messagegroups": {
       "singular": "messagegroup",
       "plural": "messagegroups",
+      "description": "A group of message definitions that are semantically related, such as a set of messages that are part of the same event flow or that share a common protocol or envelope format. One or both of the envelope and protocol definitions must be set to a value other than `None`.",
```

Comment:
we actually don't have a "description" field here in our meta-model. I've open an issue to discuss adding it